### DISCUSSION TITLE
How to use with the nuxt3?

### DISCUSSION THREAD

**Discussion by banocean** [skyra-project/discord-components#266] [Q&A]
^^^

**Comment by favna** [comment]
This should answer your question: https://github.com/skyra-project/discord-components/tree/feat/rewrite-to-lit/packages/core#sample-code-when-using-vite
Furthermore you'll want to ensure that the components are only rendered on the client and not the server. Until the Lit Element rewrite is done this library doesn't properly support Server Side Rendering because StencilJS has some pretty crappy hydration
